src/cloud/config_manager/controllers/server.go

```diff
@@ -164,6 +165,20 @@ func (s *Server) GetConfigForVizier(ctx context.Context,
 	if pemMemoryLimit == "" {
 		pemMemoryLimit = pemMemoryRequest
 	}
+
+	// We make slight modifications to the YAMLs depending on K8s version, to maintain support for older versions.
+	useOldPDB := false
+	if in.K8sVersion != "" {
+		// policy/v1beta1 podDisruptionBudget was deprecated as of v1.21.
```

Comment:
I think it's better to phrase this as `PodDisruptionBudget` graduated from beta to stable as of v1.21